src/main/java/gov/nasa/jpl/mbee/mdk/docgen/DocGenUtils.java

```diff
@@ -120,13 +125,25 @@ public static String getIndented(String name, int depth) {
     public static String htmlToXmlEntities(String html) {
         StringBuffer stringBuffer = new StringBuffer();
         Matcher matcher = ENTITY_PATTERN.matcher(html);
-
+        
+        while (matcher.find()) {
+            String replacement = null;
+        	if ( matcher.group(1).equals("&nbsp;")) {
+        		replacement = "&#160;"; //line 1766 and others of DocGen.mdzip works with this except the line 402
+      	}
+        	else
+        		replacement = htmlEntityToXmlEntity(matcher.group(1));
+            matcher.appendReplacement(stringBuffer, "");
+            stringBuffer.append(replacement);
+        }
+    	
+        /*
```

Comment:
Let's delete instead of commenting out. We have the history in Git and it is hard to diff otherwise.

Reply:
done